### ISSUE TITLE
DLSS 3.1.30 Dev DLL Now Available

### ISSUE THREAD

**Issue by Austinzveare** [emoose/DLSSTweaks#83]
I can confirm it works with DLSSTweaks but don't really know what improvements have been made. The file size has increased quite a bit though: https://github.com/NVIDIA/DLSS/raw/main/lib/Windows_x86_64/rel/nvngx_dlss.dll

**Comment by emoose** [comment]
Yep noticed it earlier too, there's 2 different versions of 3.1.30 around, seems github is the older one while a slightly newer one was included in the NV dev SDK: https://www.reddit.com/r/nvidia/comments/15a7mjt/dlss_super_resolution_sdk_3130_released_nvngx/jtj0own/

Like mentioned in that post it seems 3.1.13+ added a new UltraQuality preset, wish I looked into 3.1.13 more when it first released now 😅 

The large filesize increase is interesting too, guessing maybe a new preset was added? (maybe the preset G is now populated?)

Hopefully can look into it some more soon.

**Comment by Loganius** [comment]
It works quite bad to be honest, all games say this:

[2023-07-27 20:58:12.435] [warning] nvngx_dlssg: DisableDevWatermark failed, couldn't locate watermark string inside module
[2023-07-27 20:58:12.306] [warning] nvngx_dlss: failed to apply DLSS preset override hooks, recommend enabling OverrideAppId instead

So it has problems changing presets and fails to hide the watermark that is in 3.1.30 Frame Generation.

**Comment by PweSol** [comment]
The latest version doesn't really shimmer anymore compared to previous ones in many scenarios. It would be great to have it supported for the quality improvements at least if not for anything else.

**Comment by Githubber954** [comment]
3.1.30 seems to be forcing preset F when quality scaling is set above 0.67 like 0.77 or 0.9. Went back to 3.1.11 and preset D can be used at 0.77.
